web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1713,6 +1713,8 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   @volatile var cometGetTimeout = 140000
 
+  @volatile var putAjaxFnsInContainerSession = false
```

Comment:
I've just been making the var be the feeder for a lazy val that gets computed and reused. See [`lockedSecurityRules`](https://github.com/lift/framework/blob/joe-ajax-fns-in-session/web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala#L284).